src/workerd/api/actor.h

```diff
@@ -69,19 +71,19 @@ class DurableObjectId: public jsg::Object {
 };
 
 // Stub object used to send messages to a remote durable object.
-class DurableObject: public Fetcher {
+class DurableObject: public WorkerRpc {
```

Comment:
nit: Can this be `final` while you're here?

Reply:
https://github.com/cloudflare/workerd/compare/2e76dbfa910f6ebc6f11f1585c65ba5ad0b01462..54b85d24528f9013d34a864f241c8e771fd8358d